x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/last_value.tsx

```diff
@@ -98,6 +105,7 @@ export interface LastValueIndexPatternColumn extends FieldBasedIndexPatternColum
   operationType: 'last_value';
   params: {
     sortField: string;
+    showArrayValues?: boolean;
```

Comment:
We either need a migration for that for existing "last value" operations (because up until now it would show array values) or we need to flip the meaning of this flag (`ignoreArrayValues` or something like this).

Reply:
Makes sense 👍

Reply:
Done.